lib/ReactViews/Tools/ItemSearchTool/BackButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useTheme } from "styled-components";
+import { GLYPHS, StyledIcon } from "../../Icon";
+
+const BoxSpan: any = require("../../../Styled/Box").BoxSpan;
```

Comment:
Because Box is a `.jsx` typescript compiler does not correctly infer its type and complains about invalid props. So I use `require` so that we can cast it as any. This has unfortunately become a pattern in other places too. Ideally we convert all of them to `tsx`, but that should be a different PR.